app/models/manageiq/providers/vmware/infra_manager/vm.rb

```diff
@@ -24,7 +24,5 @@ def cloneable?
     true
   end
 
-  def supports_snapshots?
```

Comment:
I don't think this should have been deleted ... looks like it's causing failing tests.